contrib/release/post-release.sh

```diff
@@ -65,6 +65,21 @@ main() {
 
     git checkout -b pr/$version-digests $version
     ${DIR}/pull-docker-manifests.sh "$@"
+
+    echo -e "$ersion\n" > $version-release-summary.txt
+    # Grab the release notes for the current release, stop before the next
+    # release. The start of line `## vX.Y.Z` lines will match in command.
+    tail -n+4 CHANGELOG.md | sed '/^## v.*$/q' >> $version-release-summary.txt
```

Comment:
I tested this sed expression with both stable branch changelogs as well as prereleases, and it seems to correctly pick up the most recent changelog correctly. This relies on `start-release.sh` calling `prep-changelog.sh` and commiting the `CHANGELOG.md` into the release commit.